.packit.yaml

```diff
@@ -0,0 +1,37 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
```

Comment:
Thanks for referencing docs.

While this YAML may work, to be grammatically correct the file should begin with a `---` on the first line.  This is just a nit - no need to completely rebuild just for this small change.

Reply:
done